threeML/plugins/FermiLATLike.py

```diff
@@ -277,31 +300,45 @@ def __init__(
         # Activate inner minimization by default
         self.setInnerMinimization(True)
 
+        if source_name is not None:
```

Comment:
Is this 'if statement really needed? It looks like you can always set self._source_name = source_name

Reply:
Good point. Thanks!